charts/astronomer/templates/istio/sidecars.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.global.istioEnabled }}
```

Comment:
@schnie I emergency deployed this change manually using kubectl apply during the change window on 1/23/20